### PR TITLE
Add show policy package command for Checkpoint automation

### PR DESCRIPTION
**Pull request by morita-ntt**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
None

## Description
A few sentences describing the overall goals of the pull request's commits.

Checkpoint firewall runs in the combination of a SmartCenter manager and Enforcer gateway sensors. The SmartCenter can have multiple policy-packages and manage multiple gateway sensors. In the Playbook, It requires to have a task to lookup which gateway sensor installed with which policy-package.

!checkpoint-packages-list command ( get all the policy-package name from the SmartCenter) exists, but !checkpoint-package-list does not. What the latter command does is that look up gateway sensors name using policy-package name. In order not to install a wrong policy package to customer's gateway sensor( this could be a terrible disaster), The playbook needs to check the mapping of policy-package and sensor gateway name, before policy install execution.

There is no this function in the XSOAR, We added python cord to take this effect. But please put this in the next release of checkpoint automation.
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Kate-Pang** [comment]
@adi88d , thanks for the effort to provide the demo session. 
Further clarification to the request. On top of the current release CP Firewall v2 version. NTT would like to add additional command per attached :

[checkpoint-package-list (2).txt](https://github.com/demisto/content/files/5423249/checkpoint-package-list.2.txt)

@Arsenikr , @anara123  FYI.

**Review by anara123 — APPROVED** [review]

**Review by adi88d — APPROVED** [review]

**Comment by Kate-Pang** [comment]
Hi @adi88d , can you assist us to go through the error in circleci? Able to access circleci but would like some guidance how to fix the error? 
cc: @morita-ntt

File: Packs/CheckpointFirewall/Integrations/CheckPointFirewall_v2/CheckPointFirewall_v2.yml [hunk 1]
```diff
@@ -2088,6 +2088,40 @@ script:
     - contextPath: CheckPoint.Object.ipv4
       description: IPv4 address of the specified object.
       type: String
+    - name: checkpoint-package-list
```
Comment (anara123) [inline Packs/CheckpointFirewall/Integrations/CheckPointFirewall_v2/CheckPointFirewall_v2.yml:2091]:
```suggestion
  - name: checkpoint-package-list
```

**Comment by anara123** [comment]
@Kate-Pang @morita-ntt 
The error
```
CheckPointFirewall_v2.py:1449:14: error: "Client" has no attribute
"list_package"; maybe "list_packages"?  [attr-defined]
        result = client.list_package(identifier)
                 ^
Found 1 error in 1 file (checked 1 source file)
```
We are missing `list_package` method in the `Client` class. Seems that not all the code was committed

**Comment by adi88d** [comment]
> Hi @adi88d , can you assist us to go through the error in circleci? Able to access circleci but would like some guidance how to fix the error?
> cc: @morita-ntt

sure, i fixed the flake8 errors - there was a white space in empty line,

the error from mypy is:

CheckPointFirewall_v2.py:1449:14: error: "Client" has no attribute
"list_package"; maybe "list_packages"?  [attr-defined]
        result = client.list_package(identifier)
                 ^
Found 1 error in 1 file (checked 1 source file)


@morita-ntt please check whats the correct method name because `list_package` is not exists in client class.
if you added it please update the file

**Comment by kirbles19** [comment]
@ShirleyDenkberg Please review:
1. Release Notes (.md)
2. Integration (.yml) - new command only
3. README (.md) - any changes made to the YML make to the relevant section of the README.

**Comment by ShirleyDenkberg** [comment]
@anara123 Doc review completed.

**Comment by morita-ntt** [comment]
> @Kate-Pang @morita-ntt
> The error
> 
> ```
> CheckPointFirewall_v2.py:1449:14: error: "Client" has no attribute
> "list_package"; maybe "list_packages"?  [attr-defined]
>         result = client.list_package(identifier)
>                  ^
> Found 1 error in 1 file (checked 1 source file)
> ```
> 
> We are missing `list_package` method in the `Client` class. Seems that not all the code was committed

sorry if I do not understand correctly.
I think list_package exists
--
 def list_package(self, identifier: str):
            return self._http_request(method='POST', url_suffix='show-package',
                                      headers=self.headers, json_data={'name': identifier})

**Comment by adi88d** [comment]
@morita-ntt please sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=9498)

**Pull request merged**